### PR TITLE
Display additional tags after import

### LINKED ISSUE #7511: ImageStream doesn't match Docker hub tags
```
[danni@demersus badger-django]$ oc import-image node
Waiting for the import to complete, CTRL+C to stop waiting.
The import completed successfully.

Name:			node
Created:		3 minutes ago
Labels:			<none>
Annotations:		openshift.io/image.dockerRepositoryCheck=2016-02-22T01:06:33Z
Docker Pull Spec:	172.30.140.184:5000/badger-staging/node

Tag	Spec		Created		PullSpec									Image
0	node:0		3 minutes ago	node@sha256:f143f081250865a7162c136a87411175883e14999c6a16e671df9fd3484b732b	
4.0.0	node:4.0.0	3 minutes ago	node@sha256:c4bf9f9b8a159b0bb22788cc9faa56b8dec8c95f41928619b4712ec13e8c7e0d	
5.0.0	node:5.0.0	3 minutes ago	node@sha256:68674ff38dea31567a05ce0c5129c211a2efc41ee2d36a66f9c160a1fe5f7e94	
5.6.0	node:5.6.0	3 minutes ago	node@sha256:efdd15393df6574213c3efdfedf2040e6d9bb6715304573954ce05955642c59e	
latest	node:latest	3 minutes ago	node@sha256:9ff3bc4ab9d8f825446b2fe31bc8eba358eb47e258ae6042b4fdbf9194e30f07	
```

Which is not what is on https://hub.docker.com/r/library/node/tags/

### PR DESCRIPTION
**Pull request by smarterclayton**
Needs cleanup in the future for legibility (multi-column printout).

Fixes #7511 [test]

### CODE REVIEW

**Comment by smarterclayton** [comment]
Prune e2e started failing.

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
@fabianofranz @soltysh review

**Comment by fabianofranz** [comment]
LGTM

File: pkg/cmd/cli/cmd/importimage.go [hunk 1]
```diff
@@ -243,6 +244,10 @@ func RunImportImage(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, arg
 		}
 
 		fmt.Fprintln(out, info)
+
+		if r := result.Status.Repository; r != nil && len(r.AdditionalTags) > 0 {
+			fmt.Fprintf(out, "\ninfo: The remote repository contained %d additional tags which were not imported: %s\n", len(r.AdditionalTags), strings.Join(r.AdditionalTags, ", "))
```
Comment (soltysh) [inline pkg/cmd/cli/cmd/importimage.go]: Shouldn't we inform the user why they weren't imported?

**Comment by soltysh** [comment]
Small nit, but generally LGTM

File: pkg/cmd/cli/cmd/importimage.go (hunk 1 above)
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/importimage.go]: "For performance" is the best we can do.  When I come back to cleanup
the output into columns I'll add that

**Comment by smarterclayton** [comment]
[merge]

File: pkg/cmd/cli/cmd/importimage.go (hunk 1 above)
Comment (soltysh) [inline pkg/cmd/cli/cmd/importimage.go]: SGTM

**Comment by jwforres** [comment]
bindata issue should be fixed [merge]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**